tox.ini

```diff
@@ -89,7 +89,8 @@ commands = {posargs}
 basepython = python3
 deps = -r{toxinidir}/build-requirements.txt
 commands =
-    charmcraft build
+    charmcraft clean
+    charmcraft -v pack
```

Comment:
can you add a "charm clean" too after the `charmcraft pack`?, that way we don't have container images lingering around.

Reply:
Yes, `charmcraft clean` is correct.

Reply:
I have added the ```charmcraft clean``` command as requested.